util/keyfile/src/config.rs

```diff
@@ -1,48 +1,69 @@
 // Copyright (c) 2018-2021 The MobileCoin Foundation
 
-use rand::{rngs::StdRng, SeedableRng};
-use std::path::PathBuf;
+use std::{cmp, fs, path::PathBuf};
+
 use structopt::StructOpt;
 
+// Hack to work around Vec special handling in structopt
+type VecBytes = Vec<u8>;
```

Comment:
Is this still necessary when you specify `parse()` in the structop/clap args?

Reply:
I think so? Maybe I was doing it wrong, but it was complaining when I was using Vec<u8> directly.